Content.Client/GameObjects/Components/Cargo/CargoConsoleBoundUserInterface.cs

```diff
@@ -27,6 +28,8 @@ public class CargoConsoleBoundUserInterface : BoundUserInterface
         public string BankName { get; private set; }
         [ViewVariables]
         public int BankBalance { get; private set; }
+        [ViewVariables]
+        public Tuple<int,int> ShuttleCapacity { get; private set; }
```

Comment:
Using a full class like this is way too verbose. Use value tuples with named fields instead:

```cs
public (int currentOrder, max) ShuttleCapacity { get; private set; }
```

Way simpler to read and also more efficient because it doesn't need to allocate a class and such.

Reply:
Will do